vrecord

```diff
@@ -554,17 +554,17 @@ _get_avfoundation_inputs(){
     done < <("${FFMPEG_BIN}" -nostdin -hide_banner -f avfoundation -list_devices 1 -i dummy 2>&1 | grep -A 10 "AVFoundation audio devices" | grep -o "\[[0-9]\].*" | cut -d " " -f2-)
 }
 
-_get_avfctl_inputs(){
-    # set avfctl input options
-    unset AVFCTL_DEVICES
-    while read avfctl_device ; do
-        AVFCTL_DEVICES+=("${avfctl_device}")
+_get_dvrescue_inputs(){
+    # set dvrescue input options
+    unset DVRESCUE_DEVICES
+    while read dvrescue_device ; do
```

Comment:
I think this needs to be moved into an `else` clause so it doesn't run if the system is Linux (at least until I have time to make changes to the code to switch Linux side over to dvrescue as well. As written this displays the device name detected via dvrescue in the options (yay! that's ultimately a good thing) but is misleading to users until dvrescue is integrated as currently linux vrecord can only capture from the default DV device

Reply:
changing order to something like the following seems to work on linux for now:

```
_get_dvrescue_inputs(){
    # set dvrescue input options
    if [[ "${OS_TYPE}" = "linux" ]] ; then
        DVRESCUE_DEVICES+=('Default DV Device')
    else
        unset DVRESCUE_DEVICES
        while read dvrescue_device ; do
            DVRESCUE_DEVICES+=("${dvrescue_device}")
        done < <(dvrescue -list_devices | grep "\[DV\]")
        if [[ "${#DVRESCUE_DEVICES[@]}" = 1 ]] ; then # default to first input if only one
            DVRESCUE_INPUT_CHOICE="${DVRESCUE_DEVICES[0]}"
        fi
    fi
}
```

Reply:
The latest update does a lot more with getting linux to grab the device id and use it. Can you retest.